geoips/commandline/geoips_list.py

```diff
@@ -25,6 +25,105 @@
 from geoips import interfaces
 
 
+class GeoipsListSourceNames(GeoipsExecutableCommand):
+    """List Command for listing out source_names from available reader plugins.
+
+    This command is used to easily expose what source_names are supported by each
+    reader. If a reader module is missing the source_name attribute, then that will be
```

Comment:
source_names attribute?

Reply:
This has been updated to be `source_names` so that we can have multiple sources for some readers.